bench_sample.py

```diff
@@ -0,0 +1,34 @@
+import cupy as cp
+import numpy as np
+import pytest
+
+from utils import make_frame
+
+
+@pytest.fixture(params=[0, 1], ids=["AxisIndex", "AxisColumn"])
+def axis(request):
+    return request.param
+
+
+@pytest.fixture(params=[10000, 100000000], ids=["size10K", "size100M"])
+def df(request):
+    size = request.param
+    return make_frame(ncols=10, nkey_cols=0, nrows=size)
```

Comment:
Out of scope for this PR, but as you've mentioned before we definitely need to think about how to make more useful standard fixtures/generators.

Reply:
For this specific example, I think the reason why we can't use it is that we want to test dataframe that has more rows than the one in conftest. I wonder if it's ok to do a rough categorization of dataframes into "df_io_bound", "df_interim", "df_compute_bound" etc, based on how much the dataframe saturates the GPU. Apparently this assumes the problem doesn't relate - but for most simple operation it might be a reasonable assumption.

Reply:
Yeah we probably either need a handful of reusable fixtures like that or a single function that we can use to generate them.